code/game/gamemodes/changeling/powers/fakedeath.dm

```diff
@@ -17,11 +17,14 @@
 	if(user.stat != DEAD)
 		user.emote("deathgasp")
 		user.tod = worldtime2text()
-	spawn(LING_FAKEDEATH_TIME)
-		if(user && user.mind && user.mind.changeling && user.mind.changeling.purchasedpowers)
-			user << "<span class='notice'>We are ready to regenerate.</span>"
-			user.mind.changeling.purchasedpowers += new /obj/effect/proc_holder/changeling/revive(null)
+
 	feedback_add_details("changeling_powers","FD")
+
+	sleep(LING_FAKEDEATH_TIME)
+	if(user && user.mind && user.mind.changeling && user.mind.changeling.purchasedpowers)
+		user << "<span class='notice'>We are ready to regenerate.</span>"
+		user.mind.changeling.purchasedpowers += new /obj/effect/proc_holder/changeling/revive(null)

```

Comment:
This would cause the proc to block and not run the stuff below this while it slept.

These spawns were most likely spawns for a reason.